Rakefile

```diff
@@ -1,16 +1,22 @@
-require "bundler/gem_tasks"
+# frozen_string_literal: true
+
+require 'bundler/gem_tasks'
```

Comment:
The `aws-sdk-ruby` repo appears to prefer single quotes, so I guess it makes sense to switch.  But, just FYI, Rails itself and many projects in the Rails ecosystem prefer double quotes ([reference](https://github.com/rubocop-hq/rubocop/issues/5306#issuecomment-354449447)).

Reply:
Thanks for the reference! That was an interesting read and some good insight into what the community prefers. I prefer double quotes myself but I'm more keen on taking whatever the defaults are (which is currently single quotes, as well as 80 line characters). Our team manages many repos (with the bigger one being aws-sdk-ruby) so having defaults makes my life a bit easier in some fashion.